graviton/models.py

```diff
@@ -2,9 +2,10 @@
 from enum import Enum, auto
 from typing import List, Optional, Sequence, Union
 
+from algosdk.transaction import OnComplete
```

Comment:
Thx! Can, we sort these?

Reply:
yes, I was simply spamming code now, will sort the order.